lib/sda/service_test.go

```diff
@@ -444,6 +449,34 @@ func TestServiceBackForthProtocol(t *testing.T) {
 	assert.Equal(t, resp.Val, 10)
 }
 
+func TestClient_Send(t *testing.T) {
```

Comment:
Where did you see any non-CamelCase code there ? 
The golang codebase use everywhere CamelCase, these are the conventions ...